### ISSUE TITLE
Backup offerings events are not present in this view 

### ISSUE THREAD

**Issue by kiranchavala** [apache/cloudstack#9465]
ISSUE TYPE

Bug/UI

COMPONENT NAME

Component: UI, Events

CLOUDSTACK VERSION

Cloudstack version 4.19.1

SUMMARY

Events related to system vm's are not shown

Steps to reproduce  the issue 

1. Have a cloudstack environment with Veeam integration

2. Navigate to Service offerings > Backup offering > Import backup offering 

![Screenshot 2024-07-29 at 3 23 12 PM](https://github.com/user-attachments/assets/47d12100-f3b2-4a0e-ba0e-c89aef00433d)

3. Navigate to the events tabs > No events are present 

![Screenshot 2024-07-24 at 4 47 51 PM](https://github.com/user-attachments/assets/58806715-9f7f-4024-956d-60b7c8d23374)

4. The events are generated fine  (UI >Events )

![Screenshot 2024-07-24 at 4 48 06 PM](https://github.com/user-attachments/assets/2df11d3a-f943-4d58-92e3-877d304cdd00)

Actual Behaviour 

The backup.import.offering event are not present in the view (backup offering > events)

Expected behaviour 

The backup.import.offering event should be present in the view (backup offering > events)

**Comment by rohityadavcloud** [comment]
Were they logged earlier @kiranchavala ? Since backup offerings are related to the VM (and not backups themselves) it may not be an issue; same would be worth checking for vm and volume snapshots events.

**Comment by weizhouapache** [comment]
@kiranchavala @rohityadavcloud 
I am not sure if this is a bug

I checked the other service/disk/network/vpc offerings, they do not have any events for creation/import either.

**Comment by kiranchavala** [comment]
@rohityadavcloud the backup related events are logged fine, the only issue is with  displaying them in the correct UI path

When a instance is assigned to a backup offering , the event is not present in the compute >Instance > events

<img width="1177" alt="Screenshot 2024-07-29 at 4 59 59 PM" src="https://github.com/user-attachments/assets/db429000-3344-423c-9994-921bf0e9f501">

<img width="1190" alt="Screenshot 2024-07-29 at 5 05 31 PM" src="https://github.com/user-attachments/assets/fa3d18ba-4f18-4613-8098-25512d2e6770">


**The only event that is logged in the compute >Instance > events  is the "BACKUP.CREATE and BACKUP.RESTORE"**

<img width="1161" alt="Screenshot 2024-07-29 at 5 06 04 PM" src="https://github.com/user-attachments/assets/3180f552-eb52-4c67-b37a-1a68924a05df">


<img width="1180" alt="Screenshot 2024-07-29 at 5 11 00 PM" src="https://github.com/user-attachments/assets/14c77ba5-4ea1-47fe-a892-0d4e66bf2b95">

<img width="1167" alt="Screenshot 2024-07-30 at 10 44 47 AM" src="https://github.com/user-attachments/assets/397d92a5-f595-4e19-97ae-2816f60d554a">




If we look at similar pages 

service offering >compute offering > events  are present in the view

<img width="1171" alt="Screenshot 2024-07-30 at 10 31 43 AM" src="https://github.com/user-attachments/assets/fde4a3ee-2406-4ba9-8c05-1288adc23c16">


I believe the   BACKUP.IMPORT.OFFERING events should also be present in the  Service offerings > Backup offering > event 


The following events  can also be present in the compute >Instance > events

BACKUP.OFFERING.ASSIGN
BACKUP.OFFERING.REMOVE

BACKUP.SCHEDULE.CONFIGURE

**Comment by weizhouapache** [comment]
@kiranchavala 
I think the events you listed are for vms, not for backup offering.
maybe we should not display the tab `Events` for service/disk/network/backup offerings.

**Comment by rohityadavcloud** [comment]
Agree, events tabs are misleading when not for the specific resource. In this case, the backup events for a VM primarily.
